### ISSUE TITLE
losses

### ISSUE THREAD

**Issue by suuperstone** [suuperstone/issue#3]

**Comment by suuperstone** [comment]
arcFace, mv-arcFace,. curricularFace

https://emkademy.medium.com/angular-margin-losses-for-representative-embeddings-training-arcface-2018-vs-mv-arc-softmax-96b54bcd030b

**Comment by suuperstone** [comment]
Pytorch-metric-learning

loss 그냥 import 해서 편하게 사용하기 좋음
https://kevinmusgrave.github.io/pytorch-metric-learning/
